packages/preferences/src/browser/preferences-tree-widget.ts

```diff
@@ -450,7 +454,8 @@ export class PreferencesEditorsContainer extends DockPanel {
 
         let uri = preferenceUri;
         if (preferenceUri.scheme === UserStorageUri.SCHEME && homeUri) {
```

Comment:
we should get rid of homeUri, it is already encapsulated in `getConfigDirUri` and should not leak here, especially if we plan to change it.

Reply:
No, it is a different thing: https://github.com/eclipse-theia/theia/issues/7215

> it is already encapsulated in

Well no, we do not want to return with `~/.yourNameGoesHere` or get a relative path from this location, but always from user home. I think you are overlooking it

Reply:
ok, got it